src/css/controls/imports/display.less

```diff
@@ -16,7 +16,9 @@ display icons
 */
 
 .jw-display {
-    display: table;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
```

Comment:
You're going to need to test these changes on IE11. Use the views page and browserstack extensions to make it easy to check your local changes:

<img width="768" alt="Screen Shot 2021-03-10 at 11 45 49 AM" src="https://user-images.githubusercontent.com/333258/110665431-5c62fa00-8196-11eb-9e5d-7263d61f3e99.png">

Reply:
Tested in Browserstack and on a device and the styling still looks fine. Also spot checked on mobile and it looked good.
From browserstack:
<img width="1428" alt="Screen Shot 2021-03-10 at 1 05 43 PM" src="https://user-images.githubusercontent.com/26205421/110678391-42c8af00-81a4-11eb-9c82-aaedb2505753.png">

Reply:
Make sure you check all breakpoints by resizing the window. (looks like I was on 2 or 3).

Reply:
Take a closer look above. Not all screens are fine.